### PR TITLE
Added Playfair Cipher

### PR DESCRIPTION
**Pull request by Govind516**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.

### CODE REVIEW

**Review by siriak — CHANGES_REQUESTED** [review]
The code looks good, could you please add some JUnit tests? (see https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms)

**Comment by Govind516** [comment]
okay. I will add JUnit Tests.

**Comment by Govind516** [comment]
could you please check now as changes meets the requirements.

File: src/main/java/com/thealgorithms/ciphers/PlayfairCipher.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+    private char[][] matrix;
+    private String key;
+
+    public PlayfairCipher(String key) {
+        this.key = key;
+        generateMatrix();
+    }
+
+    public static void main(String[] args) {
```
Comment (siriak) [inline src/main/java/com/thealgorithms/ciphers/PlayfairCipher.java:15]: Please remove `main`, it's not needed

**Review by siriak — CHANGES_REQUESTED** [review]
Please fix the failing linter check
image

**Comment by Govind516** [comment]
Removed the _main_ function and also solved Clang format. Please check now.

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**